src/vm/runners/cairo_runner.zig

```diff
@@ -192,7 +204,7 @@ pub const CairoRunner = struct {
     /// - `entrypoint:` The address, relative to the program segment, where execution begins.
     pub fn initState(self: *Self, entrypoint: usize) !void {
         self.initial_pc = self.program_base;
-        self.initial_pc.addUintInPlace(entrypoint);
+        self.initial_pc.?.addUintInPlace(entrypoint);
```

Comment:
Is it safe to unwrap, do we need a try here maybe?

Reply:
No it is not safe indeed but this function is not completed and tested (previously we had potential `undefined` here). So I just replaced by an optional type and as a default I put `.?.` because we need to handle this error properly. Will open a PR to complete and test this function

Reply:
It is also planned to modify that as a second part of https://github.com/keep-starknet-strange/ziggy-starkdust/issues/248.